ega-data-api-dataedge/src/test/java/eu/elixir/ega/ebi/dataedge/service/internal/RemoteFileServiceImplTest.java

```diff
@@ -120,7 +126,36 @@ public void testGetFile() {
         } catch (Exception e) {
             fail("Should not have thrown an exception");
         }
-
+    }
+    
+    /**
+     * Test class for
+     * {@link RemoteFileServiceImpl#getFile(Authentication, String, String, String, String, long, long, HttpServletRequest, HttpServletResponse)}.
+     * Verify code should return NoContentException if file encryption Algorithm is gpg.
+     */
+    @Test(expected = NoContentException.class)
+    public void testGetGPGFile() {
```

Comment:
The name of the test is not intuitive. Javadocs are not the best way of expressing what you are testing as they do not appear in the reports. Substitute the name for something that explain what the test do or what do you expect to happen. Additionally there is a junit anotation to have a more descriptive text about the purpose of the test.